app/javascript/react/screens/App/Overview/components/Migrations/MigrationsCompletedList.js

```diff
@@ -84,32 +169,48 @@ const MigrationsCompletedList = ({
                   </ListView.InfoItem>
                 ]}
                 actions={
-                  (failed && (
-                    <Button
-                      onClick={e => {
-                        e.stopPropagation();
-                        retryClick(plan.href, plan.id);
-                      }}
-                      disabled={loading === plan.href}
-                    >
-                      Retry
-                    </Button>
-                  )) ||
-                  (!failed && <div style={{ width: 50 }} />)
+                  !archived && (
+                    <React.Fragment>
+                      {failed && (
+                        <Button
+                          onClick={e => {
+                            e.stopPropagation();
+                            retryClick(plan.href, plan.id);
+                          }}
+                          disabled={loading === plan.href}
+                        >
+                          {__('Retry')}
+                        </Button>
+                      )}
+                      <Button
+                        onClick={e => {
+                          e.stopPropagation();
+                          showConfirmModalAction(confirmModalOptions);
+                        }}
+                      >
+                        {__('Archive')}
+                      </Button>
+                    </React.Fragment>
+                  )
                 }
               />
             );
           })}
         </ListView>
       ) : (
         <OverviewEmptyState
-          title={__('No Migration Plans Completed')}
+          title={
+            archived
+              ? __('No Archived Migration Plans')
+              : __('No Completed Migration Plans')
+          }
           iconType="pf"
           iconName="info"
           description={
             <span>
-              {__(
-                'There are no existing migration plans in a Completed state.'
+              {sprintf(
+                __('There are no existing migration plans in %s state.'),
+                archived ? 'an Archived' : 'a Completed'
               )}
```

Comment:
The `archived` conditional needs to decide what message to display...

So if `archived` is true, then 
```js
sprintf(__('There are no existing migration plans in an Archived state.'))
```
and likewise the other message for `archived` = false

The only reason why I'm suggesting this change is because not all languages work like English.
For e.g. in a different language, the sentence construction rules or the grammar may not be the same as English and the `%s` placeholder in that particular spot may not work.

Reply:
Updated :)

I squashed my commit to help those working on the G backport

Reply:
Cool! :) 
That looks good!